core/network/fabric-interop-cc/contracts/interop/membership.go

```diff
@@ -134,12 +513,27 @@ func verifyMemberInSecurityDomain(s *SmartContract, ctx contractapi.TransactionC
 	if err != nil {
 		return fmt.Errorf("Failed to unmarshal membership: %s", err.Error())
 	}
+	return verifyMemberInSecurityDomain2(s, ctx, certPEM, cert, membership, requestingOrg)
+}
+
+// verifyMemberInSecurityDomain2 function verifies the identity of the requester according to
+// the Membership for the external network the request originated from.
+// This takes a decoded X.509 certificate as argument (and optionally the certificate in PEM format too).
+// It takes a membership structure as argument.
+func verifyMemberInSecurityDomain2(s *SmartContract, ctx contractapi.TransactionContextInterface, certPEM string, cert *x509.Certificate, membership *common.Membership, requestingOrg string) error {
+    err := isCertificateWithinExpiry(cert)
```

Comment:
Identation issue.
Also am I missing something, or you haven't used `certPEM` at all in this function...
I think you intended to decode that for certificate when its not empty right?